.gitignore

```diff
@@ -18,4 +18,4 @@ eth/protocol
 
 # Our compiled binary
 livepeer
-livepeer_cli
+/livepeer_cli
```

Comment:
What does this do?

Reply:
Without the slash, Git was ignoring new files added to `cmd/livepeer_cli`. I believe the slash just tells Git to only ignore files/directories with the name `livepeer_cli` in the base directory.